smart-contract-example/package.json

```
@@ -22,9 +22,9 @@
   "version": "1.0.0",
   "description": "",
   "repository": "https://github.com/kurtosis-tech/awesome-kurtosis",
-  "author": "Tedi Mitiku <tedi.mitiku@kurtosistech.com>",
+  "author": "Tedi Mitiku <tedi.mitiku@kurtosis.com>",
```

hey @adschwartz this is wrong
```suggestion
  "author": "Tedi Mitiku <tedi.mitiku@kurtosistech.com>",
```